plugins/authentication/cookie/cookie.php

```diff
@@ -115,8 +116,9 @@ public function onUserAuthenticate($credentials, $options, &$response)
 
 		// Remove expired tokens
 		$query = $this->db->getQuery(true)
-			->delete('#__user_keys')
-			->where($this->db->quoteName('time') . ' < ' . $this->db->quote(time()));
+			->delete($this->db->quoteName('#__user_keys'))
+			->where($this->db->quoteName('time') . ' < :now')
+			->bind(':now', time(), ParameterType::INTEGER);
```

Comment:
` 'time' varchar(200) NOT NULL,`
```suggestion
			->bind(':now', time());
```